packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -78,7 +78,7 @@
 RESET_COUNT_START = 0
 VALUE_NOT_PROVIDED = object()
 # tolerance in seconds for new blocks not having arrived yet
-BLOCKS_STALL_TOLERANCE = 15
+BLOCKS_STALL_TOLERANCE = 60
```

Comment:
I noticed that 15 seconds are not enough for some edge cases and unnecessarily triggers the fallback mechanism.